save-frontend/src/jsMain/resources/locales/en/profile.json

```diff
@@ -1,17 +1,17 @@
-{

```

Comment:
strange that GitHub shows it as a change

Reply:
+1